packages/compass-import-export/src/modules/export.js

```diff
@@ -362,19 +370,23 @@ const fetchDocumentCount = async (dataService, ns, query) => {
  *
  * @api public
  */
-export const openExport = ({ namespace, query, count: maybeCount }) => {
+export const openExport = ({
+  namespace,
+  query,
+  count: maybeCount,
+  aggregation,
+}) => {
   return async (dispatch, getState) => {
     track('Export Opened');
     const {
       dataService: { dataService },
     } = getState();
-
     try {
       const count =
         maybeCount ?? (await fetchDocumentCount(dataService, namespace, query));
```

Comment:
Not great that we are relying there on the fact that we pass zero as a aggregation count to avoid this fetch to happen. This creates an interface for the event that is hard to reason about. If we don't want to fetch document count for aggregation, we should be explicit about it